docs/README.md

```diff
@@ -12,3 +12,177 @@ We currently have the following components available for React applications:
 -   [Score Card](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/ScoreCard.md)
 -   [Spacer](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/Spacer.md)
 -   [User Menu](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/UserMenu.md)
+
```

Comment:
I assume this will eventually live in a separate document and not be in the top level readme (just linked to), right?